devfile.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: 1.0.0
+metadata:
+  generateName: che-devfile-registry-
+
```

Comment:
what about enabling multi-root workspace ?

Reply:
will give a try :) maybe add it to https://github.com/eclipse/che/issues/19305